crates/re_space_view_spatial/src/mesh_loader.rs

```diff
@@ -23,32 +25,32 @@ impl LoadedMesh {
     ) -> anyhow::Result<Self> {
         // TODO(emilk): load CpuMesh in background thread.
         match mesh {
-            AnyMesh::Asset(asset3d) => Self::load_encoded_mesh(name, asset3d, render_ctx),
+            AnyMesh::Asset(asset3d) => Self::load_asset3d(name, asset3d, render_ctx),
             AnyMesh::Mesh(mesh3d) => Ok(Self::load_mesh3d(name, mesh3d, render_ctx)?),
         }
     }
 
-    // TODO(#3354): more like load_asset3d_parts (or just merge the two..?)
-    pub fn load_raw(
+    pub fn load_asset3d_parts(
         name: String,
-        format: MeshFormat,
+        media_type: &MediaType,
         bytes: &[u8],
         render_ctx: &RenderContext,
     ) -> anyhow::Result<Self> {
         re_tracing::profile_function!();
 
-        let mesh_instances = match format {
-            MeshFormat::Glb | MeshFormat::Gltf => {
-                re_renderer::importer::gltf::load_gltf_from_buffer(
-                    &name,
-                    bytes,
-                    ResourceLifeTime::LongLived,
-                    render_ctx,
-                )
-            }
-            // TODO(cmc): support obj
-            MeshFormat::Obj => anyhow::bail!(".obj files are not supported yet"),
+        let media_type: MediaType = media_type.as_str().parse()?;
```

Comment:
This looks wrong. We take a `MediaType` and parse it as a another `MediaType` O.o